Add Powershell DSC troubleshooting section (#1145)
OctopusDeploy/docs: docs/infrastructure/deployment-targets/windows-targets/azure-virtual-machines/via-an-arm-template-with-dsc.md

    --template-file "arm-template.json" \
    --parameters "@arm-template.parameters.json"
```

> ### Troubleshooting 

Reviewer: ```suggestion
## Troubleshooting 
```

To troubleshoot the installation, you can use [`Start-Transcript`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.host/start-transcript?view=powershell-7.1) to write the Powershell session to a file.

> In addition to this, if you're able to remote on the machine in question to find out the state of DSC, these two commands may offer diagnostic information:

Reviewer: ```suggestion
In addition, if you have remote access to the machine you are troubleshooting installation for, these two commands may offer diagnostic information about the state of DSC:
```
